libs/market-info/src/components/market-info/info-key-value-table.tsx

```diff
@@ -40,8 +40,9 @@ const Row = ({
       return value;
     }
     if (decimalPlaces) {
-      return `${parseFloat(
```

Comment:
this breaks market-info target stake and supplied stake values.

Reply:
NOTE: I will handle the follow up on ALL number formatters :D https://github.com/vegaprotocol/frontend-monorepo/issues/2179